partiql-conformance-tests/build.rs

```diff
@@ -0,0 +1,296 @@
+use codegen::{Formatter, Function, Scope};
+use ion_rs::value::reader::{element_reader, ElementReader};
+use ion_rs::value::{Element, Sequence, Struct, SymbolToken};
+use ion_rs::IonType;
+use std::ffi::OsStr;
+use std::fs;
+use std::fs::File;
+use std::io::Write;
+use std::path::{Path, PathBuf};
+use walkdir::WalkDir;
+
+const PASS_PARSE_TESTS_DIR: &str = "partiql-tests/partiql-test-data/pass/parser";
+const FAIL_PARSE_TESTS_DIR: &str = "partiql-tests/partiql-test-data/fail/parser";
+const PARSER_IMPORT: (&str, &str) = ("partiql_parser", "lalr_parse");
+
```

Comment:
There are some things that you can `println` in main here to [direct cargo](https://doc.rust-lang.org/cargo/reference/build-scripts.html#outputs-of-the-build-script) how and when to rerun your build script.